docs/application/web/api/6.5/device_api/mobile/tizen/ml.html

```diff
@@ -213,16 +216,24 @@ <h2>1. Type Definitions</h2>
             <li>
 MVNC - Intel Movidius Neural Compute SDK (libmvnc).            </li>
             <li>
-NNFW - Neural Network Inference framework, which is developed by SR (Samsung Research).            </li>
+NNFW - Neural Network Inference Framework, which is developed by Samsung Research.            </li>
```

Comment:
I see many references to Samsung Research, are these NPUs published as Samsung-specific?
Should we just says Samsung?

@mijin85cho your thoughts?

Reply:
I have same question with @annie-samsung .
Is there any reason to specify "Samsung Research"? Or is it acceptable to mark it as "Samsung"?

Reply:
The descriptions are consistent with the descriptions of related native functions, which also has the phrase "Samsung Research", refer to file:
./docs/application/native/api/mobile/6.0/group__CAPI__ML__NNSTREAMER__PIPELINE__MODULE.html lines 716, 752. I would rather leave it as it is now.